credmark/cmf/types/position.py

```diff
@@ -1,11 +1,25 @@
+import credmark.cmf.model
 from credmark.dto import DTO, DTOField, cross_examples
 from .token import Token, NativeToken
+from .price import Price
 
 
 class Position(DTO):
     amount: float = DTOField(0.0, description='Quantity of token held')
     asset: Token
 
+    def get_value(self):
+        context = credmark.cmf.model.ModelContext.current_context()
+        try:
```

Comment:
@DHG18 Actually you can remove this exception handler. Any exception here would be some kind of temporary error so we don't want to catch it because then incorrect results would be cached in our db. So we just let an exception fail.